### PR TITLE
Create ufr-tours-histoire-de-lart-note.csl

### PR DESCRIPTION
**Pull request by Emic37**
Add new style based on tapuscrit. Good for contemporary art history student at the university of Tours.

### CODE REVIEW

File: ufr-tours-histoire-de-lart-note.csl [hunk 1]
```diff
@@ -0,0 +1,662 @@
...
+      </if>
+      <else>
+        <text variable="issue"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="collection">
+    <text variable="collection-title" quotes="true" prefix="coll. "/>
+  </macro>
+  <citation et-al-min="4">
```
Comment (rmzelle) [inline ufr-tours-histoire-de-lart-note.csl]: @Emic37, we don't allow `et-al-min` to be used without setting `et-al-use-first` on the same element.

Comment (Emic37) [inline ufr-tours-histoire-de-lart-note.csl]: thank you very much!

**Comment by rmzelle** [comment]
@Emic37, does your university department describe this style format on a website or in a printed document we could link to?

**Comment by rmzelle** [comment]
Hi, we're closing this issue because we haven't heard from you in a long time. Note that we can always reopen this issue if you get back to us.

**Pull request closed without merging**